summarize-anthropic/README.md

```diff
@@ -0,0 +1,27 @@
+# Summarize PDFs with Anthropic
+
+We can use Claude2, which has a long context window, to summarize PDFs.
+
+To do this, we can use various promps from LangChain hub, such as:
+
+* [Fun summarization prompt](https://smith.langchain.com/hub/hwchase17/anthropic-paper-qa)
+* [Chain of density summarization prompt](https://smith.langchain.com/hub/lawwu/chain_of_density)
+
+Claude2 has a large (100k token) context window, allowing us to summarize documents over 100 pages.
+
+# Invoke
+
+You can call this summarization template.
+
+TO DO: Add clarification on how to spin this up.
+
+```
+summarization_chain = RemoteRunnable('localhost:8000/anthropic-summarization')
```

Comment:
@efriis we should add clarification / instruction on how to set up / run RemoteRunnable.

Reply:
Oh the import is just `from langserve.client import RemoteRunnable`!

Larger docs are in the works I think. There's a breaking change in an upcoming release to support error messaging, and then it'll be more stable.

Reply:
Ah fair there are a few more steps:

1. You'll want to add `http://` to the url
2. You'll have to specify that this package needs to be installed in langserve as `/anthropic-summarization`. You might want to change to `summarize-anthropic` because that'll be the default (it's the package name from pyproject.toml by default iirc)

Reply:
OK, cool. updated README. we can walk through this flow w/ one example.